conftest.py

```diff
@@ -0,0 +1,2 @@
+import sideboard
```

Comment:
This file is relevant for unit tests, though at the moment we haven't written any for this repo.